src/Jackett.Common/Definitions/letseed.yml

```
@@ -0,0 +1,215 @@
+    - {id: 27, cat: Other, desc: "TV+"}
+    - {id: 36, cat: PC, desc: "Program"}
+    - {id: 37, cat: PC, desc: "İşletim Sistemi"}
+    - {id: 38, cat: PC/Mac, desc: "Mac"}
+    - {id: 39, cat: PC, desc: "Windows"}
+    - {id: 40, cat: TV/Sport, desc: "Spor"}
+    - {id: 41, cat: TV/Sport, desc: "Spor Sunumları"}
+    - {id: 17, cat: Other, desc: "VIP Sunumlar"}
+    - {id: 18, cat: Other, desc: "VIP Sunumlar"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, imdbid]
+    movie-search: [q, imdbid]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Filter freeleech only
+    default: false
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: added
+    options:
+      added: created
+      seeders: seeders
+      size: size
+      name: title
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+  - name: info_tpp
+    type: info
+    label: Results Per Page
+    default: For best results, change the <b>Torrents per page:</b> setting to <b>40</b> on your account profile. The Default is <i>30</i>.
+  - name: info_activity
+    type: info
+    label: Account Inactivity
+    default: "To keep records updated regularly, all inactive accounts will be deleted after 60 days of inactivity."
+
+login:
+  path: login.php
+  method: form
+  form: form[action="takelogin.php"]
+  captcha:
+    type: image
+    selector: img#regimage
+    input: imagestring
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+    logout: ""
+  error:
+    - selector: '[http-equiv="refresh"][content*="/login.php"]'
+      # TODO: This tracker uses `http-equiv="refresh"` for all login form submissions,
+      # valid or invalid, and uses the `error=#` query parameter to tell the subsequent
+      # login form what error message to display. How should this definition yield a
+      # useful error message?
+  test:
+    path: browse.php
+    selector: a[href*="/logout.php"]
+    # TODO: While debugging false login success for this definition, I read in the docs
+    # that a redirect is considered as failure and that `selector:` is only required for
+    # web UIs that render the login form directly in the response to `path:`. For all
+    # the previous definitions I've worked on *and* the examples I copied for them, the
+    # web UIs responded with redirects but the definitions include `selector:`. Are
+    # those existing definitons wrong in this aspect?  If no, why use `selector:` for
+    # web UIs that redirect to the login form?
+
+download:
+  before:
+    path: takethanks.php
+    method: post
+    inputs:
+      torrentid: >-
+        {{ re_replace .DownloadUri.AbsolutePath "/download-torrent-([0-9]+).*" "$1"}}
```

```suggestion
      torrentid: "{{ .DownloadUri.Query.id }}"
```
the download block has a handy variable we can use, see https://github.com/Jackett/Jackett/wiki/Definition-format#download-variables

---

Well, I worked off the Prowlarr docs, but they concur with the Jackett docs you linked. Are you saying that somehow the `.Query` sub-key somehow accesses path elements. Can you clarify? Where is that documented in that link? Sorry if I'm being blind.